misc/rhcos-layer/Makefile

```diff
@@ -11,9 +11,12 @@ TEE ?= tdx
 build: #check-podman get-pull-secret get-ocp-release-image ## Default target: Build the RHCOS layer
 	@echo "Building RHCOS layer..."
 	if [ "$(TEE)" == "tdx" ]; then \
-		podman build --authfile $(REGISTRY_AUTH_FILE) --build-arg OCP_RELEASE_IMAGE=$(shell cat ./.ocp-release-image) -t $(IMAGE_TDX) -f tdx/Containerfile . ;\
+		podman build --authfile $(REGISTRY_AUTH_FILE) --build-arg OCP_RELEASE_IMAGE=$(shell cat ./.ocp-release-image) \
+		-t $(IMAGE_TDX) -f tdx/Containerfile . ;\
 	else [ "$(TEE)" == "snp" ]; \
-		podman build --authfile $(REGISTRY_AUTH_FILE) --build-arg OCP_RELEASE_IMAGE=$(shell cat ./.ocp-release-image) -t $(IMAGE_SNP) -f snp/Containerfile . ;\
+		podman build --authfile $(REGISTRY_AUTH_FILE) --build-arg OCP_RELEASE_IMAGE=$(shell cat ./.ocp-release-image) \
+		--build-arg CA_CERTS_FILE=$(CA_CERTS) --build-arg REGISTRY_AUTH_FILE=$(REG_AUTH) \
```

Comment:
CA_CERTS and REG_AUTH can be used for TDX as well. No ?

Reply:
I think so as we are going to use the same initrd. I modified it only for SNP after modifying the Containerfile under snp/. There exists a separate Containerfile under tdx/ which I did not modify. Do you think the rhcos layered image for tdx will be same as that for snp?

Reply:
Broadly the Containerfile for tdx will be same except the packages which will still come from CentOS stream. Rest of the customizations for initrd etc remains the same for both snp and tdx

Reply:
We can also do it in a separate PR. I